semanticdb/javac/src/main/scala/scala/meta/internal/semanticdb/javac/semantics/Elements.scala

```diff
@@ -178,6 +178,7 @@ trait Elements {
             }
           }
           s.TypeSignature(
+            typeParameters = Some(s.Scope()),
```

Comment:
I think we should consider using the `scalapb.no_box = true` option, that would avoid further problems like this. Worth squeezing into v4?

Reply:
Let's open a ticket and defer to post-v4? If we do no_box here, maybe we'd want to do no_box in other places, too, and it'd be great to do that kind of change at once.

Reply:
I opened https://github.com/scalameta/scalameta/issues/1741